plugins/dev-scripts/config/webpack.config.js

```diff
@@ -109,29 +92,55 @@ module.exports = (env) => {
             },
           ],
           include: [resolveApp('./src/'), resolveApp('./test/')],
+        }
+    );
+  }
+  // The following rules are always in effect.
+  rules.push(
+      // Load Blockly source maps.
+      {
+        test: /(blockly\/.*\.js)$/,
+        use: [require.resolve('source-map-loader')],
+        enforce: 'pre',
+      },
+      // Run babel to compile both JS and TS.
+      {
```

Comment:
Instead of forking the script with a conditional, you can instead follow a similar pattern to how we determine whether or not to run the TS checker plugin.

eg: 

```
[
 !env.skipLint && { ...}
].filter(Boolean)
```

Reply:
Genius, done, and tested the same way and it all still works.